docs/en/guides/sre/s3-multi-region.md

```diff
@@ -0,0 +1,183 @@
+---
+sidebar_label: Replicating a single shard across two AWS regions using S3 Object Storage
+---
+
+# Replicating a single shard across two AWS regions using S3 Object Storage
+
+- [Deploy ClickHouse](#deploy-clickhouse)
+- [Configure ClickHouse Keeper](#configure-clickhouse-keeper)
+- [Create S3 Buckets](#create-s3-buckets)
+- [Define a cluster](#define-a-cluster)
+- [Configure networking](#configure-networking)
+- [Testing](#testing)
+
+## Deploy ClickHouse
+
+Deploy ClickHouse on three hosts, in the sample configurations these are named `chnode1`, `chnode2`, and `keepernode1`.  `chnode1` and `chnode2` run ClickHouse and ClickHouse Keeper.  `keepernode1` is only running ClickHouse Keeper.
+
+Place `chnode1` in one AWS region, and `chnode2` in a second.  `keepernode1` can be placed in either of the two regions, maybe in a different availability zone within the region?
```

Comment:
Yes, better in another availability zone, so if one AZ becomes unavailable, both regions remain available for writes, because it requires two of three keeper nodes to remain available.